python/cuml/benchmark/runners.py

```diff
@@ -59,10 +59,15 @@ def _run_one_size(
         else:
             cpu_elapsed = 0.0
 
+        speedup = cpu_elapsed / float(cu_elapsed)
+        if verbose:
```

Comment:
for verbose, nice to mention algo_pair.name in the output since it's easy sometimes to lose track when doing many runs